controller/forgotpassword.js

```diff
@@ -0,0 +1,96 @@
+const dotenv = require('dotenv').config()
+const UserSignup = require('../model/signupschema');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const transporter = require('../utils/nodeMailer');
+
+
+
+
+const fgproute = async(req,res,next)=>{
+
+    const userid = req.userid;
+    const userdetail = await UserSignup.findOne({_id:userid});
+    const hashedpassword = userdetail.password
+    if(!userid){
+        res.status(500).json({message:"Something went wrong"})
```

Comment:
send a proper error message whenever possible

Reply:
ok sir but  I have read somewhere that we should not send exact same error because of security reasons.